guides/v2.1/cloud/live/live-sanity-check.md

```diff
@@ -36,7 +36,7 @@ Your Git branch must have the following files for building and deploying for you
 
 ## Test build your code locally before pushing {#test-build}
 
-Sometimes you just want to test your build prior to pushing your code to Git. You can use a specific set of commands to build locally. Do **NOT** push the generated build files from this test to your remote Git branch. This is a trial run to ensure no issues occur before pushing to Git. Remember, when you push to the remote Git branch, a full build and deploy process begins automatically.
+Sometimes you just want to test your build prior to pushing your code to Git. You can use a specific set of commands to build locally. Do **NOT** push the generated build files from this test to your remote Git branch. This is a test to ensure no issues occur before pushing to Git. Remember, when you push to the remote Git branch, a full build and deploy process begins automatically.
```

Comment:
Sometimes you just want to ... is a little fuzzy --   consider making it more directive -- You can use Magento CLI commands to build and test your site before pushing your code to git.

Reply:
Yeah, I didn't write that, for sure. All I did here was remove the word "trial".